src/pages/Home.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import '../App.css';
+import NewListButton from '../components/NewListButton';
+import * as ls from 'local-storage';
+/*import FirebaseTesting from './FirebaseTest';*/
+import fb from '../lib/firebase';
+import { Link, Redirect } from 'react-router-dom';
+import FooterTabs from '../components/FooterTabs';
+/*import ShoppingList from './ShoppingList';
+import AddItem from './AddItem';*/
+
+function Home({token, setToken}) {
+	const [inputToken, setInputToken] = useState('');
+	/*const [items, setItems] = useState([]);*/
+	// Adds Value of token entered into inputToken
+	const handleChange = e => {
+		setInputToken(e.target.value);
+	};
+	// Checks the firestore db to see if inputToken exists
+	const checkToken = e => {
+		e.preventDefault();
+		let db = fb.firestore();
+		db.collection(`${inputToken}`)
+			.get()
+			.then(data => {
+				if (data.empty === true) {
+					alert('Bad token, Try again or Create a New List');
+				} else {
+					ls.set('ShoppingListToken', inputToken);
+					setToken(inputToken);
+				}
+			});
+	};
+
+	return (
+		<div className='App'>
+			<Link to='/FirebaseTesting'>FirebaseTesting</Link>
+			{token ? (
+				<Redirect to='/ShoppingList' />
+			) : (
+				<div className='Home-token'>
+					<input
+						type='text'
+						name='inputToken'
+						placeholder='enter token'
+						onChange={handleChange}
```

Comment:
I'd add a 'required' for this field, so that if it is empty and you join the list, it will require the user for a token.